l4v-deploy/steps.sh

```diff
@@ -38,12 +46,18 @@ then
   staging-manifest ssh://git@github.com/seL4/verification-manifest.git
   echo "::endgroup::"
 else
+  if [ -n "${INPUT_MANIFEST}" ]; then
+    declare -a MANIFEST_FILE=("--manifest-file" "${INPUT_MANIFEST}")
+  else
+    declare -a MANIFEST_FILE
+  fi
```

Comment:
This is  a fairly bash-specific way. We can currently assume bash on the Ubuntu runners, so I'm fine with leaving it in, but if you have a POSIX way of achieving the same, we should probably use that.

(I don't remember a good POSIX way of doing this with multiple parameters where `INPUT_MANIFEST` could contain spaces, so no problem if you want to keep it).

Reply:
The only thing I can think of is to inline the `if` with the `seL4-pp` call. With only one such option, it seems ok, though it obviously wouldn't scale to multiple options.

Reply:
Let's keep it for now. It was just in case you had a solution I didn't know about.